buildsettings.yaml

```diff
@@ -23,6 +23,6 @@ dev:
   ubiDseImages:
     - datastax/dse-server:6.8.4-ubi7
   ubiOssImages:
-    - datastax/cassandra:3.11.7-ubi7
+    - datastax/cassandra:3.11.6-ubi7
```

Comment:
3.11.7-ubi7 is not in dockerhub, so this was breaking integ tests when loading images

Reply:
cc @bradfordcp